course_content/notebooks/numpy_intro.ipynb

```diff
@@ -24,7 +24,7 @@
    "source": [
     "## Workshop Objectives\n",
     "\n",
-    "At the end of this workshop you will be able to:\n",
+    "The aim of this workshop is to enable you to:\n",
```

Comment:
this appears to be the only concern and I prefer using the wording i have used. 

So, i'm going to adopt this for now, as I'm presenting the trial run of this course